source/frontend/src/features/properties/map/project/add/models.ts

```diff
@@ -1,23 +1,50 @@
-import { Api_Project } from 'models/api/Project';
+import { Api_Product, Api_Project } from 'models/api/Project';
 import { NumberFieldValue } from 'typings/NumberFieldValue';
 import { toTypeCode } from 'utils/formUtils';
 
+export class ProductForm {
```

Comment:
Would this fall into our new category of needing a unit test?

Reply:
It definitely would. I think once the fromApi is set it would be good to add it